### ISSUE TITLE
NTR: Renal biopsy: fibronectin deposits (Lab: pathology: biopsies & morphology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9301]
New term request Renal biopsy: fibronectin deposits (elthat_130331035207):

**Comment by pnrobinson** [comment]
@MickeySegal
What part of the kidney are the deposits localized to?

**Comment by MickeySegal** [comment]
https://www.omim.org/entry/137950
"There were homogeneous PAS-positive deposits in the mesangium and subendothelial space that showed strong immunoreactivity to fibronectin...   Electron microscopy showed massive fine granular deposits with a homogeneous distribution. Immunohistochemistry showed positive staining of fibronectin, as well as IgG and fibrinogen, in the mesangium and capillary loop. "

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000428
- label: Glomerular fibronectin deposits
- parents: [Glomerular deposits (HP:0030949)](https://hpo.jax.org/app/browse/term/HP:0030949)
- pmid: PMID:35774511
- def: Accumulation of fibronectin in the glomerulus.
- comment: Fibronectin glomerulopathy (FNG) is characterized by extensive deposition of fibronectin in the glomeruli, especially in the mesangial and subendothelial regions.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9301
- synonyms:
